.readthedocs.yaml

```diff
@@ -0,0 +1,19 @@
+# Read the Docs configuration file for Sphinx projects
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+version: 2
+
+# Set the OS, Python version and other tools you might need.
+# OS can only be: ubuntu-20.04, ubuntu-22.04
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3.9"
```

Comment:
I think you can use a more recent version, e.g. 3.11

Reply:
Done